tests/store/test_file_store.py

```diff
@@ -208,18 +223,24 @@ def test_get_run(self):
                 run_info.pop("metrics")
                 run_info.pop("params")
                 run_info.pop("tags")
+                # Add special case for lifecycle_stage since it doesn't belong in the serialized
```

Comment:
Nit: Could we update this now that lifecycle_stage is in the serialized run info?